providers/os/resources/os.lr.manifest.yaml

```diff
@@ -355,6 +375,22 @@ resources:
       source: {}
       target: {}
     min_mondoo_version: 5.15.0
+  ipv4Address:
+    fields:
+      broadcast: {}
+      cidr: {}
+      gateway: {}
+      ip: {}
+      subnet: {}
+    min_mondoo_version: 9.0.0
+  ipv4Addresss:
```

Comment:
This looks like a typo

Reply:
interesting, I did make a typo once, but I fixed it, I guess the code generation doesn't remove this on its own? 🤔 

Good catch!

Reply:
Yep they only add. never remove